packages/valory/contracts/offchain_aggregator/contract.yaml

```diff
@@ -18,5 +18,9 @@ class_name: OffchainAggregatorContract
 contract_interface_paths:
   ethereum: build/VerySimpleOffchainAggregator.json
 dependencies:
+  eth-abi:
+    version: ==2.2.0
```

Comment:
Why is this required by `autonomy check-packages` after `0.3.4`? This is not used anywhere in the repo. Only from `web3`.

Reply:
Might be an error in the logic. Suggest you open an issue. Only direct imports should be tagged

Reply:
Opened https://github.com/valory-xyz/open-aea/issues/427.